src/components/dashboard/Amount.js

```diff
@@ -146,30 +270,71 @@ const Amount = (props: AmountProps) => {
               unit={token}
             />
           </Section.Stack>
-          <Section.Row>
-            <Section.Row grow={1} justifyContent="flex-start">
-              <BackButton mode="text" screenProps={screenProps}>
-                {t`Cancel`}
-              </BackButton>
-            </Section.Row>
-            <Section.Stack grow={3} style={styles.nextButtonContainer}>
-              <NextButton
-                nextRoutes={
-                  isBridge
-                    ? ['SendLinkSummary', 'Home']
-                    : isReceive
-                    ? ['Reason', 'ReceiveSummary', 'TransactionConfirmation']
-                    : isNativeFlow
-                    ? ['SendToAddress', 'SendLinkSummary']
-                    : ['Reason', 'SendLinkSummary', 'TransactionConfirmation']
-                }
-                canContinue={handleContinue}
-                values={{ ...params, ...restState, amount: GDAmountInWei, ...bridgeState }}
-                disabled={loading}
-                {...props}
-              />
+          {isSend && (
```

Comment:
have you tested it with the native flow ? set REACT_APP_DELTA=true in Your local env, at dev instance you will be able to switch onto Goerli and select gEth to send. If you need some Goerli pls ask me

Reply:
yes I did test, have not tested sending any funds yet tho. will do